go/oasis-node/cmd/node/unsafe_reset.go

```diff
@@ -34,15 +34,17 @@ var (
 		Run:   doUnsafeReset,
 	}
 
+	runtimesGlob = filepath.Join(runtimeRegistry.RuntimesDir, "*")
+
 	nodeStateGlobs = []string{
 		"abci-mux-state.*.db",
 		"persistent-store.*.db",
 		tendermint.StateDir,
-		runtimeRegistry.RuntimesDir,
+		filepath.Join(runtimesGlob, "history.db"),
```

Comment:
maybe should reference `runtime/history/.DbFilename` (currently it's not exported though), instead of hardcoding `"history.db"`

Reply:
Good idea, done.